scripts/linux/install.sh

```
@@ -70,6 +70,7 @@ curl -sSfL "https://github.com/gitleaks/gitleaks/releases/download/v${GITLEAKS_V
 
 ## pre-commit
 uv tool install pre-commit
+cd "${ROOT_DIR}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)